code/modules/mob/living/simple_animal/abnormality/teth/my_sweet_home.dm

```diff
@@ -0,0 +1,96 @@
+//Brain go brrrr.
+/mob/living/simple_animal/hostile/abnormality/my_sweet_home
+	name = "My Sweet Home"
+	desc = "This cozy little house is a safe nest built only for you. Everything is here for you..."
+	icon = '---'
+	icon_state = "---"
+	icon_living = "---"
+	icon_dead = "---"
+	threat_level = TETH_LEVEL
+	can_breach = TRUE
+	del_on_death = TRUE
+	var/icon_aggro = "---"
+	var/lc13admin_mode_variable = 0
```

Comment:
What is it for?

Reply:
I mostly meant line 13. It doesn't seem to be doing anything important and doesn't get changed to TRUE anywhere in the code.

Reply:
From the forsaken murderer file:
_var/lc13admin_mode_variable = 0 //for variables that are edited during a round._